core/src/main/java/org/apache/calcite/rel/rules/FilterProjectTransposeRule.java

```diff
@@ -60,7 +62,7 @@
    * {@link org.apache.calcite.rel.core.Correlate} from being de-correlated.
    */
   public static final FilterProjectTransposeRule INSTANCE =
-      new FilterProjectTransposeRule(Filter.class, Project.class, true, true,
+      new FilterProjectTransposeRule(LogicalFilter.class, LogicalProject.class, true, true,
```

Comment:
This is a backwards incompatible change. Please add an entry in the history.md (see breaking changes).

Reply:
Would it make sense (to keep backwards compatibility) to leave the rule's "default" `INSTANCE` as it is (matching Filter+Project), and create a separate e.g. `LOGICAL_INSTANCE` matching LogicalFilter+LogicalProject?

Reply:
This is a good idea. I will try that. Thanks @rubenada